src/plots/mapbox/layout_attributes.js

```diff
@@ -39,19 +39,38 @@ var attrs = module.exports = overrideAll({
         description: [
             'Sets the mapbox access token to be used for this mapbox map.',
             'Alternatively, the mapbox access token can be set in the',
-            'configuration options under `mapboxAccessToken`.'
+            'configuration options under `mapboxAccessToken`.',
+            'Note that accessToken are only required when `style`',
```

Comment:
I think there's a line missing here... "when `style` ... something?"

Reply:
The full text is:

https://github.com/plotly/plotly.js/blob/d81f28cf985f5ec912b83d60b0bb0b066665a6bb/src/plots/mapbox/layout_attributes.js#L40-L45

Reply:
ah ok, i was thrown off by the parenthetical in the middle of the sentence. Works.